### PR TITLE
Code review fixes

### PR DESCRIPTION
**Pull request by ntorbinskiy**

### CODE REVIEW

File: src/js/modules/activityModule.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    if (!isTaskCompleted(taskIcon.querySelector("svg").dataset.svgsPath)) {
+      return;
+    }
+
+    const taskTime = taskItem.querySelector("span.activity_time");
+
+    const taskName = taskItem.querySelector(".task_content").textContent;
+
+    const taskText = taskItem.querySelector(".text");
```
Comment (1alexvash) [inline src/js/modules/activityModule.js:72]:
```suggestion
    const taskTime = taskItem.querySelector("span.activity_time");
    const taskName = taskItem.querySelector(".task_content").textContent;
    const taskText = taskItem.querySelector(".text");
```
They are related so group them together

File: src/js/modules/activityModule.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        taskItem,
+        taskTime,
+        taskName,
+      };
+
+      const noPoints = createNoPoints(noPointsOptions);
+
+      taskText.after(noPoints);
+
+      return;
```
Comment (1alexvash) [inline src/js/modules/activityModule.js:89]: This returns nothing does this statement serve any purpose?

File: src/js/modules/projectModule.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+import {
+  createTotalPoints,
+  findTotalPointsElement,
+  updateTotalPointsScore,
+} from "../components/totalPoints";
+
+const linkLogic = () => {
+  const listOfTasks = document.getElementsByClassName("items");
+
+  nodeToArray(listOfTasks).map((task) =>
```
Comment (1alexvash) [inline src/js/modules/projectModule.js:12]: If the ES6 explicit return does not fit a single line, then wrap it with {} and add a return statement to indicate the zone of function

File: src/js/modules/projectModule.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      const buttonLinkParent = taskChildNode.querySelector(
+        ".task_list_item__actions--active"
+      )?.childNodes;
+
+      if (!buttonLinkParent) {
+        return;
+      }
+
+      taskChildNode.addEventListener("mouseenter", () => {
+        if (buttonLinkParent[0].className !== "button-href") {
```
Comment (1alexvash) [inline src/js/modules/projectModule.js:23]: 0 is a magic number, what's the reason we select first button?

File: utils/build.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import webpack from "webpack";
+import config from "../webpack.config.js";
+
+const { chromeExtensionBoilerplate, ...configWithoutBoilerplate } = config;
+
+webpack(configWithoutBoilerplate, (err, stats) => {
+  if (err) {
```
Comment (1alexvash) [inline utils/build.js:7]: use full names

**Review by 1alexvash — APPROVED** [review]
Looks nicer

File: src/js/modules/activityModule.js (hunk 2 above)
Comment (ntorbinskiy) [reply src/js/modules/activityModule.js:93]: No, I've deleted it

File: src/js/modules/projectModule.js (hunk 4 above)
Comment (ntorbinskiy) [reply src/js/modules/projectModule.js:23]: Because our custom button is first in list, but yeah-this is a magic number, I'll change it to `querySelector`

**Pull request merged**